firefox/ScreenSharing/index.js

```diff
@@ -0,0 +1,41 @@
+var pageMod = require("sdk/page-mod");
+var gDomains = ['*.example.com'];
+//var {Cc, Ci, Cu} = require("chrome");
+
+var allowDomainsPrefKey = 'media.getusermedia.screensharing.allowed_domains';
+var prefsService = require("sdk/preferences/service")
+
+exports.main = function (options, callbacks) {
+
+  if (options.loadReason === 'install' || 'enable') {
+    var allowDomainsPrefKey = 'media.getusermedia.screensharing.allowed_domains';
+
+    var curPref = prefsService.get(allowDomainsPrefKey);
```

Comment:
We don't need a `.split(',')` here, because the following lines examine the `curPref` string (not an array).